docs/content/api-reference.md

```diff
@@ -21,143 +21,35 @@ These are the bare bones of a Spectacle presentation, the two most essential tag
 
 Wraps the entire presentation and carries most of the overarching slide logic, like theme and template context.
 
-| Props    | Type                        | Example |
-| -------- | --------------------------- | ------- |
-| theme    | [Custom Theme Object](TODO) | `TODO`  |
-| template | [Layout Render Prop](TODO)  | `TODO`  |
+| Props    | Type                                       |
+| -------- | ------------------------------------------ |
+| theme    | [Styled-system theme object](TODO)         |
```

Comment:
Where will this link eventually come form?

Reply:
The theme page. Eventually.